src/api/spec/models/user_spec.rb

```diff
@@ -278,10 +278,26 @@
         expect(user.email).to eq('some@email.com')
       end
     end
+  end
+
+  describe 'create_user_with_fake_pw' do
+    it_behaves_like 'successful login'
```

Comment:
It doesn't make sense to use this here since that shared example calls `create_user_with_fake_pw!`